contrib/test/integration/e2e-base.yml

```diff
@@ -61,3 +61,19 @@
 
 - name: Enable iptables NAT for the bridge
   command: sysctl -w net.bridge.bridge-nf-call-iptables=1
+
+- name: Change /var label
+  command: 'semanage fcontext -a -R -t container_file_t /var/'
+  when: e2e_selinux_enabled
+
+- name: Reload /var label
+  command: 'restorecon -v /var/'
+  when: e2e_selinux_enabled
+
+- name: Change /etc label
+  command: 'semanage fcontext -a -R -t container_file_t /etc/'
```

Comment:
You can do similar for /etc, but I think this is dangerous because other programs that are running on the system might want to read files in /etc like /etc/passwd and they will not be able to read them if /etc/passwd is labeled as container_file_t.

Reply:
What AVC's are you seeing if you don't do the change of label for /etc?

Reply:
I am not seeing any AVCs anymore, but earlier on e2e_features_* runs I was seeing AVC denials for the name "etc-hosts" which I assumed was /etc/hosts. I then checked openshift nodes for the labelling there, and it seems as though all of /etc is labeled container_file_t, so I tried that.